webapp/src/main/resources/templates/frontend/notification-settings.html

```diff
@@ -10,63 +11,129 @@
 <body>
     <section layout:fragment="content">
         <div id="notification-settings-container">
-            <h2 class="h5 mt-3">Regular email notifications</h2>
+            <h1 class="h5 mt-3">Notification Settings</h1>
             <hr class="border-secondary"/>
-            <fieldset class="mb-3">
-                <p>
-                    You can get email notifications about recent or upcoming releases at a frequency of 2 or 4 weeks.
-                    These emails are always sent on Sundays.
-                </p>
-                <div class="form-group">
-                    <div class="custom-control custom-switch">
-                        <input type="checkbox" class="custom-control-input" id="notification-toggle">
-                        <label class="custom-control-label" for="notification-toggle">Regular notifications</label>
+            <p>
+                You can receive notifications about current or upcoming releases at a frequency of 2 or 4 weeks. The notifications are
```

Comment:
```suggestion
                You can receive notifications about current or upcoming releases at a frequency of two or four weeks. These notifications are
```
The numbers looks better i think. The "These" makes it clear that only the period notifications come on Sundays, the others can can be different of course.